sbt-plugin/src/main/scala/ch/epfl/scala/SubmitDependencyGraph.scala

```diff
@@ -82,6 +82,13 @@ object SubmitDependencyGraph {
     val snapshotUrl = s"${githubApiUrl()}/repos/${githubRepository()}/dependency-graph/snapshots"
 
     val snapshotJson = CompactPrinter(Converter.toJsonUnsafe(snapshot))
+
+    val snapshotJsonFile = IO.withTemporaryFile("dependency-snapshot-", ".json") { file =>
```

Comment:
The `keepFile` argument should be set to `true` otherwise it deletes the file before returning.